### DISCUSSION TITLE
[VL] set fallback reason as WARN instead of INFO

### DISCUSSION THREAD

**Discussion by FelixYBW** [apache/gluten#6599] [General]
Fallback reason is important info and should be a WARN.

INFO printed too much lines so I usually set to WARN log level. 

@zhli1142015 @Yohahaha @ulysses-you @jackylee-ch @kecookier @WangGuangxin 
Your thought?

**Comment by Yohahaha** [comment]
👍  `spark.gluten.sql.validation.logLevel` was set to WARN by default in our product.

**Comment by FelixYBW** [comment]
We will set it to WARN since no object.

**Comment by kecookier** [comment]
I agree with your suggestion. In my situation, we will also collect fallback information and send it to Kafka for further analysis. Thank you for your input!

**Comment by FelixYBW** [comment]
#6676 set default to WARN.

I didn't note we already have a config to control this.
